savant/meta/object.py

```diff
@@ -152,8 +155,15 @@ def add_attr_meta(
                 element_name=element_name, name=name, value=value, confidence=confidence
             )
         else:
-            self._attributes[(element_name, name)] = AttributeMeta(
-                element_name=element_name, name=name, value=value, confidence=confidence
+            if not (element_name, name) in self._attributes:
+                self._attributes[(element_name, name)] = []
+            self._attributes[(element_name, name)].append(
+                AttributeMeta(
+                    element_name=element_name,
+                    name=name,
+                    value=value,
+                    confidence=confidence,
+                )
             )
```

Comment:
Maybe `ObjectMeta.add_attr_meta()` should take an `AttributeMeta` instance to avoid repacking?

Reply:
This would already be refactoring, which would entail changes to examples, documentation, etc. Let's leave it as simple bug fixing.